restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -131,7 +126,7 @@ static Object toJsonValue(final Object cqlValue) {
     if (cqlValue instanceof InetAddress) {
       return ((InetAddress) cqlValue).getHostAddress();
     }
-    if (cqlValue instanceof List || cqlValue instanceof Set) {
+    if (cqlValue instanceof Collection) {
```

Comment:
I don't think we want to catch maps or tuples here.

Reply:
Maps are not Collections, will check Tuples.

Reply:
`TupleValue`s do not implement `Collection` (or subtype) either. Will check offline if I should change this back.

Reply:
My bad, I think I got mixed up with the `isCollection` check.